ext/standard/tests/file/popen_pclose_basic2.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test popen() and pclose function
```

Comment:
What's going on with this test? It doesn't seem to be using popen() at all :)

Reply:
Oh, I wanted to keep the useful parts of the original test (`ext/standard/tests/file/popen_pclose_error.phpt`) after I extracted the tests for error conditions. I didn't notice that it tests `fopen` and `fclose`. :)